Vagrantfile

```diff
@@ -58,10 +58,19 @@ sudo apt-get install rvm -y
 
 sudo usermod -a -G rvm $USER
 
+# Setup mastodon user & home directory for using systemd:
+sudo adduser --disabled-login --gecos "" mastodon
+sudo usermod -a -G rvm mastodon
+sudo ln -nsf /vagrant/ /home/mastodon/live
+
 SCRIPT
 
 $provisionB = <<SCRIPT
 
+# Clean up previously added .bash_profile file, otherwise
+# vagrant up --provision will continuously grow this file with repeated lines
+rm -f ~/.bash_profile
```

Comment:
I noticed that on multiple `vagrant up`'s and `vagrant up --provision` calls, if the machine hadn't been destroyed first, then the `.bash_profile` file grew with repeated lines.